configs/common-cache/config.d/40-osg-caching-plugin.cfg

```diff
@@ -5,46 +5,79 @@
 # * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
 # **********************************************************************
 #
-# This file is part of the StashCache Daemon
-# https://opensciencegrid.org/docs/data/stashcache/overview/
+# This file is part of the XCache Daemon
 
+# Load the correct libraries for XCache to operate:
+# First, the proxy library itself.
+ofs.osslib  libXrdPss.so
+# Second, the cache underlying the proxy.
+pss.cachelib libXrdFileCache.so
 
-# The location of data federation this XCache participates in
-#
-# By default, we set this to the OSG Data Federation
-pss.origin redirector.osgstorage.org:1094
+if named atlas-xcache
+   pss.origin =
 
+   # Load the Rucio namespace-mapping plugin
+   pss.namelib -lfncache /usr/lib64/XrdName2NameDCP4RUCIO.so
+
+   # Specifies the names of oss.space name for data and the oss.space name for metadata
+   pfc.spaces data meta
+else
+   pss.origin redirector.osgstorage.org:1094
+
+   # Tune the client timeouts to more aggressively timeout.
+   pss.setopt ParallelEvtLoop 10
+   pss.setopt RequestTimeout 25
+   pss.setopt ConnectTimeout 25
+   pss.setopt ConnectionRetry 2
+fi
 
 # Cache plugin parameters
 #
 # These values should be sufficient for most use cases. To change them, override them
 # in a later file such as 90-local.cfg.
 
+if defined ?~XC_RAMSIZE
```

Comment:
Is this correct? Should it be `if defined ?~XC_BLOCKSIZE`? Also, why does the comment above say 'override them in a later file' if they can be set with environment vars?